Постройте мелкие сетки на гридах для подходящих компонентов

AlyonaPrikhodchenko opened the pull request:
Вопросы в телеге

---
:mortar_board: [Постройте мелкие сетки на гридах для подходящих компонентов](https://up.htmlacademy.ru/htmlcss/33/user/1875081/tasks/9)

:boom: https://htmlacademy-htmlcss.github.io/1875081-sedona-33/6/

AlyonaPrikhodchenko committed:
Доработки, модальное окно

AlyonaPrikhodchenko committed:
скрыла модалку

ad0v0 commented on styles/styles.css:
```diff
@@ -160,12 +160,16 @@ body {
-.icon-link:hover path {
-  fill:#756257;
+.icon-link:hover svg {
+  color:#756257;
```
форматирование кода, см code guide - нужно добавить пробел между значеним и двоеточием, проверить весь код

ad0v0 commented on styles/styles.css:
```diff
@@ -1389,18 +1416,112 @@ body {
+
+input[type='number'],
+input[type="number"]:hover,
+input[type="number"]:focus {
```
https://up.htmlacademy.ru/htmlcss/33/criteries#CSS-02 стилизация по глобальному тегу, использовать класс

AlyonaPrikhodchenko committed:
изменения

изменения в рэндже, кнопках фильтрации, модалке

AlyonaPrikhodchenko committed:
Продолжение модалки

AlyonaPrikhodchenko committed:
Модалка

Готовая модалка

AlyonaPrikhodchenko committed:
Готовый макет

Осталось проверить чистоту кода

AlyonaPrikhodchenko committed:
Готовый макет

Осталось отшлифовать

AlyonaPrikhodchenko commented:
<img width="158" alt="image" src="https://user-images.githubusercontent.com/88833115/155893938-c8927e97-1321-41ec-881c-bdba00a573c6.png"> все еще висит вопрос, что делать с object-fit изображений

**В общем макет уже готов, я еще просмотрю код стилей на порядок свойств, а так вот по мелочи осталось и пора отправлять на первичную проверку**

AlyonaPrikhodchenko committed:
Готовый макет

Скрыла модалку

AlyonaPrikhodchenko committed:
Финишные доработки

Отредачен код по стайлгайду, добавлена фавиконка (ранее), отредачены стили выдачи ошибки в форме и добавлены атрибуты required.

ad0v0 commented:
https://github.com/htmlacademy-htmlcss/1875081-sedona-33/pull/6#issuecomment-1053631824

мистерия 🤯

ad0v0 commented:
- http://joxi.ru/Y2LaQxKuxgwB1A давай сверим ПП. У тебя какой результат при выкл object fit contain ?
- http://joxi.ru/n2YJXEZUkOjo3A убрать состояние error по умолчанию. Его должен навешивать разработчик при реализации функционала валидации
- http://joxi.ru/52apBKvtlnGEgm предлагаю заменить на классы , которые состоят из слов add/remove, тк они более "семантические"
- http://joxi.ru/D2Pnd73CwxdJkr добавить вертикальные отступы между рядами, чтобы элементы не склеивались 
- http://joxi.ru/L21BGwXtwJ6w7A добавить текстовое описание кнопкам

AlyonaPrikhodchenko committed:
Доработки

Подогнала по пп как могла, проверила код

AlyonaPrikhodchenko commented:
Алёна, [01.03.2022 20:10]
Так, небольшой отчет:

Мои маленькие победы:
1. Код проверила, отредачила по кодгайду
2. На валидность проверила, ошибки испавила, код валидный
3. К pp макет максимально подогнала, кроме некоторых моментов
4. Активные стили всему задала
5. Исправила блоки фильтрации после просмотра лайва, стало гораздо лучше, теперь чекбоксы и радиокнопки реагируют если тыкнуть
6. Фавиконку добавила

Мои проблемы:
1. Вопрос с картинками все еще открыт, я вообще не знаю, важно ли это так, как мне кажется, может и так ок.
2. Форма отправки эмейла. Выше скрин кинула с паттерном, жду ответа, норм ли так или удалить и просто класс написать отдельный. Хотя вариант с паттерном рабочий.
3. Текст в блоке ЗАИНТЕРЕСОВАЛИСЬ? я никак не могу подогнать по макету, но по макету он кривой, а у меня все ровненько над кнопкой, так что жду твоего мнения. Посмотришь по пиксель перфект различия
4. У пагинации пришлось сделать отступ справа 2пх что-бы подогнать под пп, ну это я скорее как утверждение, чем вопрос, просто это чутка странно, я все же думаю, что дизайнер кривовато сделала макет
5. Ну и модалка. В принципе там все отлично, все на местах, тултип рабочий. Но не могу никак идеально подогнать под пп, когда меняю какие-то марджины, все как-то странно двигается, а текст в полях ввода и вовсе широкий в сравнении с макетом.

Я скинула снова на проверку дз, даже если ты ее примешь, я могу исправления сделать для следующей дз. Это сообщение продублирую в гитхаб. Осталось разобраться с этими мелочами и хотелось бы уже на первичную проверку кидать

AlyonaPrikhodchenko commented:
http://joxi.ru/Y2LaQxKuxgwB1A давай сверим ПП. У тебя какой результат при выкл object fit contain ?
**НУЖНО РЕШИТЬ ЭТОТ ВОПРОС**
http://joxi.ru/n2YJXEZUkOjo3A убрать состояние error по умолчанию. Его должен навешивать разработчик при реализации функционала валидации
**ПО ЭТОМУ ВОПРОСУ ТОЖЕ УТОЧНИЛА, ЖДУ ОТВЕТ**
http://joxi.ru/52apBKvtlnGEgm предлагаю заменить на классы , которые состоят из слов add/remove, тк они более "семантические"
**СДЕЛАЛА**
http://joxi.ru/D2Pnd73CwxdJkr добавить вертикальные отступы между рядами, чтобы элементы не склеивались
**СДЕЛАЛА**
http://joxi.ru/L21BGwXtwJ6w7A добавить текстовое описание кнопкам
**СДЕЛАЛА**

ad0v0 commented:
- http://joxi.ru/V2V73MdsBeLD3A заменить псевдокласс invalid на класс со словом -error . В дальнейшем его смогут переисопльзовать , как js , так и back-end разработчики
- http://joxi.ru/12MR5kZigOxEQA FF фон у select 
- карточки: у меня была старая превьюха, видимо, правки были внесены в макет, а превьюху я вслед не обновила в своих ресурсах. Забери пжлст изображения к карточкам или самостоятельно заново выкачай и замени существующие на новые, после этого перепроверь плс результат.
- ![hotel-1](https://user-images.githubusercontent.com/9592349/156395496-806e48bd-5807-4f88-bc0f-834c34794691.png)
- ![hotel-1 (1)](https://user-images.githubusercontent.com/9592349/156395506-672abbca-bedd-446a-ad58-c825c40c9f95.png)
- ![hotel-1 (2)](https://user-images.githubusercontent.com/9592349/156395515-696a51b0-1bfb-49d9-a766-13ce180c49da.png)
- ![hotel-1 (3)](https://user-images.githubusercontent.com/9592349/156395523-9586ef3f-ed46-4413-a471-cbafea03454f.png)

- Текст в блоке ЗАИНТЕРЕСОВАЛИСЬ - оставляем, похоже на косяк в макете
- пагинация http://joxi.ru/Dr8YPw4SKEYwXm : не похоже на ошибку дизайнера, скорее всего это было сделано специально, чтобы визуально пагинация находилась как бы внутри от левой границы начала серой лини сверху
- Модалка: ПП в норме, текст в поле стилизовать через :placeholder, нужно поправить uppercase

AlyonaPrikhodchenko committed:
Шлифовка

Убрала паттерн, добавила класс error-email вместо псевдокласса invalid, заменила картинки карточек

AlyonaPrikhodchenko committed:
Последние правки

Прописала шрифт в полях

AlyonaPrikhodchenko committed:
Вернула :invalid

Вернула псевдокласс invalid и оставила доп.класс для ошибки при этом

AlyonaPrikhodchenko committed:
Обновила название класса

Сделала класс ошибки универсальным
